src/explore-education-statistics-admin/src/components/editable/EditableSectionBlocks.tsx

```diff
@@ -82,7 +86,10 @@ const EditableSectionBlocks = (props: EditableSectionBlockProps) => {
           <div
             key={block.id}
             id={`editableSectionBlocks-${block.id}`}
-            className="govuk-!-margin-bottom-9"
+            className={classNames(
+              'govuk-!-margin-bottom-9',
+              openedCommentId.includes(block.id) && styles.openSectionBlock,
```

Comment:
Use object syntax for conditions in `classNames` ([read the docs](https://www.npmjs.com/package/classnames#usage) please)